app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -234,34 +235,37 @@ class HomeFragment : Fragment() {
             ::dispatchModeChanges
         )
 
+        val blocklistMiddleware = BlocklistMiddleware(requireContext().settings())
+        val initialState = HomeFragmentState(
+            collections = components.core.tabCollectionStorage.cachedTabCollections,
+            expandedCollections = emptySet(),
+            mode = currentMode.getCurrentMode(),
+            topSites = components.core.topSitesStorage.cachedTopSites,
```

Comment:
Looks like this is rebased incorrectly since it's missing the `.sort()`. I think it would be better undo this change where we move the `HomeFragmentState` into a variable and just keep it inline.

Reply:
Reverted this to inline, using a `run` block to try and minimize the diff